README.md

```diff
@@ -1,30 +1,53 @@
-<h1 align="center">Hi 👋, I'm Andrij</h1>
-<h3 align="center">A passionate javaScript developer from Ukraine</h3>
+# Welcome to Real-time-chat 👋
 
-<p align="left"> <img src="https://komarev.com/ghpvc/?username=andrij-maksymyshyn&label=Profile%20views&color=0e75b6&style=flat" alt="andrij-maksymyshyn" /> </p>
+This is an online chat, where you can chat with your friends.
+(It's server part of project.)
 
-- 🔭 I’m currently working on [real-time-chat](https://github.com/Andrij-Maksymyshyn/real-time-chat)
+### 🏠 [Homepage](https://github.com/Andrij-Maksymyshyn/real-time-chat)
 
-- 🌱 I’m currently learning **Node.js**
+## Install
 
-- 🤝 I’m looking for help with [Oleksij Chestnyh (mentor)](https://github.com/chestnyh)
+```sh
+yarn
```

Comment:
yarn from root dir doesn't work. You should describe here from what folder you should run in

Reply:
fixed